shell/config/router/navigation-guards/clusters.js

```diff
@@ -1,62 +1,22 @@
 import { DEFAULT_WORKSPACE } from '@shell/config/types';
-import { applyProducts } from '@shell/store/type-map';
 import { ClusterNotFoundError, RedirectToError } from '@shell/utils/error';
 import { get } from '@shell/utils/object';
 import dynamicPluginLoader from '@shell/pkg/dynamic-plugin-loader';
 import { AFTER_LOGIN_ROUTE, WORKSPACE } from '@shell/store/prefs';
-import { BACK_TO } from '@shell/config/local-storage';
 import { NAME as FLEET_NAME } from '@shell/config/product/fleet.js';
 import { validateResource, setProduct } from '@shell/utils/auth';
-import { getClusterFromRoute, getProductFromRoute, getPackageFromRoute } from '@shell/utils/router';
+import { getClusterFromRoute, getProductFromRoute, getPackageFromRoute, routeRequiresAuthentication } from '@shell/utils/router';
 
-let beforeEachSetup = false;
-
-export default async function({
-  route, store, redirect, from, $plugin, next
-}) {
-  if ( store.getters['auth/enabled'] !== false && !store.getters['auth/loggedIn'] ) {
-    return;
-  }
-
-  const backTo = window.localStorage.getItem(BACK_TO);
-
-  if (backTo) {
-    window.localStorage.removeItem(BACK_TO);
```

Comment:
I didn't port backTo over. It's non-functional in master and Neil seems keen to remove it. You can view a thread in team-container-ui where this was discussed.

I added an issue to track the removal of the rest of this feature https://github.com/rancher/dashboard/issues/11268

Reply:
Seen the conversation already 👍 
UI/UX should be approved by Ken though and not Neil.
As he did not express any opinion, I take it as a tacit agreement.